components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java

```diff
@@ -627,6 +627,8 @@ private String appendRequiredAttributes(String sqlQuery, List<String> requiredAt
                     case IdPManagementConstants.IDP_PROVISIONING:
                         sqlQuery += ", " + IdPManagementConstants.PROVISIONING + " ";
                         break;
+                    case IdPManagementConstants.IDP_GROUPS:
```

Comment:
Let's add a comment similar to https://github.com/wso2/carbon-identity-framework/blob/382f4ef74d3b7f6618dde29a35972c379156f14f/components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java#L601 for better clarity

Reply:
Fixed by - https://github.com/wso2/carbon-identity-framework/pull/5767/commits/2a27638dd5d632e6c9a817bad42efe3b3d4fc628